src/Client/HttpExceptionHandler.php

```diff
@@ -79,6 +79,21 @@ protected function getResponseMessage(ResponseInterface $response)
         $decodedBody = json_decode($responseBody->getContents(), true);
         $responseBody->rewind();
 
-        return isset($decodedBody['message']) ? $decodedBody['message'] : $response->getReasonPhrase();
+        $message = isset($decodedBody['message']) ? $decodedBody['message'] : $response->getReasonPhrase();
```

Comment:
there is already a way to get error messages (see https://api.akeneo.com/php-client/exception.html#unprocessable-entity-exception). do you really think it's needed to change this behavior? And if true, update the doc please

Reply:
![ah](https://user-images.githubusercontent.com/7206368/31944428-5b3415ce-b8cc-11e7-9300-760f676e1007.gif)
It's not needed anymore :p